stl/inc/format

```diff
@@ -2546,6 +2584,13 @@ template <class _CharT, class _OutputIt, integral _Integral>
 _NODISCARD _OutputIt _Write_integral(
     _OutputIt _Out, _Integral _Value, _Basic_format_specs<_CharT> _Specs, _Lazy_locale _Locale);
 
+template <class _CharT, class _OutputIt>
+_NODISCARD _OutputIt _Write_escaped(_OutputIt _Out, basic_string_view<_CharT> _Value, char _Delim);

```

Comment:
This overload of `_Write_escaped` (the one that doesn't take `_Basic_format_specs<_CharT> _Specs`) is being declared here, but not called until its definition. I believe we can simply drop this declaration.